### PR TITLE
Release 3.9.1

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@conda-forge/core if anyone can merge this it would be great.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  skip: true  # [(unix and py27) or win]
+  skip: true  # [(unix and py<36) or win]
```
Comment (msarahan) [inline recipe/meta.yaml]: why skip with py<36?  Is it just to reduce the build matrix?

Comment (SylvainCorlay) [inline recipe/meta.yaml]: yes

**Comment by msarahan** [comment]
I don't have rights here, but other than my question about the skip condition, looks fine to me.

**Comment by SylvainCorlay** [comment]
In next PR, I will attempt to not skip windows. This is required for cling.

**Pull request merged**